config/routes.rb

```diff
@@ -9,6 +9,8 @@
           id: guide.slug
     end
 
+    get '/locations', to: 'locations#index'
```

Comment:
Seems a bit neater, I'd originally opted for a `scope` block but it was too much for the task at hand